### PR TITLE
Theme Added

### PR DESCRIPTION
**Pull request by ergenekonyigit**
Screenshot
Screenshow

### CODE REVIEW

**Comment by nesl247** [comment]
Hi @ergenekonyigit, thanks for submitting this.

Can you please make the following changes?
- Submit a PR to https://github.com/dracula/dracula.github.io with the screenshot (you don't have to make any other changes in the PR.). For this screenshot, if possible, please close the project tree first. It would make the screenshot look a lot nicer.
- Change this README to point to the screenshot instead of imgur.
- Remove myself and @zenorocha as the maintainers from the README. We're just there as part of the template (placeholders).

Thanks again!

**Comment by ergenekonyigit** [comment]
@nesl247 
- Submitted a pr to https://github.com/dracula/dracula.github.io.
- Uploaded screenshot with closed the project tree in screenshots folder.
- Removed you and @zenorocha as the maintainers from the README.

**Comment by nesl247** [comment]
Sorry for the delay. Can you please add `screenshot.png` to this PR in the root of the repo, and then link to that instead via `Screenshot` instead?

Also can you add installation instructions in the install section instead of linking to the site?

Again, really sorry for the delays. I just got done moving and @zenorocha and I had been discussion some changes to the way we handle the repos, so things got a bit behind.

File: README.md [hunk 1]
```diff
@@ -2,19 +2,32 @@
...
+#### Install using Git
+
+If you are a git user, you can install the theme and keep up to date by cloning the repo:
+ 
+```
+$ git clone https://github.com/dracula/qtcreator.git ~/.config/QtProject/qtcreator/styles
+```
+
+#### Install manually
+Download using the <a href="https://github.com/dracula/qtcreator/archive/master.zip">GitHub .zip download</a> option and unzip them
```
Comment (nesl247) [inline README.md]: Please add a period at the end.

File: README.md [hunk 2]
```diff
@@ -2,19 +2,32 @@
...
+ 		Move the <code>dracula.xml</code> file to <code>~/.config/QtProject/qtcreator/styles</code>
```
Comment (nesl247) [inline README.md]: I'm not sure why the huge indentation here?

**Comment by nesl247** [comment]
Just two minor issues and it'll be ready for merge.

File: README.md (hunk 2 above)
Comment (ergenekonyigit) [inline README.md]:
``` this is markdown code block for clone theme.

Comment (nesl247) [inline README.md]: This comment is about line 19.

**Comment by ergenekonyigit** [comment]
@nesl247 fixed two issues.

**Pull request merged**